docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -5,10 +5,18 @@ Cassandra connector
 The Cassandra connector allows querying data stored in
 `Apache Cassandra <https://cassandra.apache.org/>`_.
 
-Compatibility
--------------
-
-Connector is tested against Cassandra version 2.2 and 3.11, but any intermediate or newer versions are expected to work.
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Cassandra
+data source are:
+
+* Cassandra version 2.2 or higher or PostgreSQL 9.6 or higher
```

Comment:
Why do we need to mention PostgreSQL here?

Reply:
Cut and paste error.. removed.